entities/storobj/storage_object.go

```diff
@@ -898,6 +898,14 @@ func UnmarshalProperties(data []byte, properties *map[string]interface{}, aggreg
 						if err != nil {
 							returnError = err
 						}
+					case jsonparser.Object:
+						nestedProps := map[string]interface{}{}
+						err := json.Unmarshal(innerValue, &nestedProps)
+						if err != nil {
+							returnError = err
+						}
+						val = nestedProps
```

Comment:
**question:** shall we break in case of error ?

Reply:
yep, I'll return now!